AllReadyApp/Web-App/AllReady.UnitTest/Areas/Admin/Controllers/CampaignAdminControllerTests.cs

```diff
@@ -82,29 +88,63 @@ public void IndexReturnsCorrectDataWhenUserIsOrgAdmin()
             controller.SetClaims(claims);
 
             ViewResult view = (ViewResult)controller.Index();
-
-            // verify the fetch was called
             mockMediator.Verify(mock => mock.Send(It.Is<CampaignListQuery>(c => c.OrganizationId == OrganizationId)));
 
             // Org admin should only see own campaigns
             IEnumerable<CampaignSummaryModel> viewModel = (IEnumerable<CampaignSummaryModel>)view.ViewData.Model;
             Assert.NotNull(viewModel);
             Assert.Equal(viewModel.Count(), 1);
             Assert.Equal(viewModel.First().OrganizationId, OrganizationId);
-            
         }
 
-
-        [Fact(Skip = "NotImplemented")]
-        public void IndexReturnsCorrectViewModel()
+        public void IndexReturnsCorrectDataWhenUserIsNotOrgAdmin()
         {
+            int OrganizationId = 99;
+            var mockMediator = new Mock<IMediator>();
+            mockMediator.Setup(x => x.Send(It.Is<CampaignListQuery>(c => c.OrganizationId == null)))
+                .Returns((CampaignListQuery q) => {
+                    // return some models 
+                    List<CampaignSummaryModel> ret = new List<CampaignSummaryModel>();
+                    ret.Add(new CampaignSummaryModel { OrganizationId = OrganizationId });
+                    ret.Add(new CampaignSummaryModel { OrganizationId = OrganizationId + 1 });
+                    return ret;
+                }
+            );
+            var mockImageService = new Mock<IImageService>();
```

Comment:
When you're testing a method that doesn't use the dependency (like IImageService) you don't have to mock it, you can just pass in null. It makes the test a little lighter, too.